src/backup/BackupPhrase.test.tsx

```diff
@@ -7,38 +7,37 @@ import BackupPhrase from 'src/backup/BackupPhrase'
 import { Screens } from 'src/navigator/Screens'
 import { createMockStore, getMockStackScreenProps } from 'test/utils'
 
-describe.each([
-  { settingsScreen: Screens.SettingsDrawer },
-  { settingsScreen: Screens.Settings },
-  {},
-])('BackupPhrase (settingsScreen: $settingsScreen)', (routeParams) => {
-  it('renders correctly with backup not completed', () => {
-    const tree = render(
-      <Provider store={createMockStore()}>
-        <BackupPhrase {...getMockStackScreenProps(Screens.BackupPhrase)} />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
-  })
+describe.each([{ settingsScreen: Screens.Settings }, {}])(
```

Comment:
actually this describe.each (added here: https://github.com/valora-inc/wallet/pull/5121/files#diff-6215a8f5c86692eac81da63fc0f0a41ace9c791f2ad68e8d17687a175d817fedR14) isn't really doing anything because `routeParams` is never passed. Just runs the same test multiple times. I am cleaning this up in a follow up PR (#5336) to add better tests